docker/Dockerfile.tpu

```diff
@@ -23,7 +23,7 @@ RUN --mount=type=cache,target=/root/.cache/pip \
     --mount=type=bind,source=.git,target=.git \
     python3 -m pip install \
         -r requirements/tpu.txt
-RUN python3 setup.py develop
```

Comment:
Why switch from this? The docs recommend this method and I use it on 3.11 for TPU

Reply:
Oh i forgot to revert this, we will just need to upgrade setuptools here.